es-design-system/pages/molecules/es-video.vue

```diff
@@ -0,0 +1,51 @@
+<!-- eslint-disable vuejs-accessibility/no-static-element-interactions -->
+<template>
+    <div>
+        <h1>
+            Video
+        </h1>
+        <p>
+            Shows the embed video when the cover is clicked.
+        </p>
+        <div>
+            <es-video
+                :alt-text="altText"
+                class="my-450 mb-200 mb-lg-0"
+                :cover-image-url="placeholderEsThumbnail"
+                :embed-url="embedUrl" />
+        </div>
+        <ds-doc-source
+            :comp-code="compCode"
+            comp-source="es-vue-base/src/lib-components/EsVideo.vue"
+            :doc-code="docCode"
+            doc-source="es-design-system/pages/molecules/es-video.vue" />
+    </div>
+</template>
+<script>
+import placeholderEsThumbnail from '@/assets/placeholder/energysage_thumbnail.png';
+
+export default {
+    name: 'EsVideoDocs',
+    data() {
+        return {
+            compCode: '',
+            docCode: '',
+            altText: 'Community solar: Everything you want to know before you join',
+            placeholderEsThumbnail,
+            embedUrl: 'https://www.youtube.com/embed/hgmZG3GLLNg',
```

Comment:
Since `altText` and `embedUrl` are only used once, I'd say it might be cleaner to move these values straight onto the component definition above, and then we won't need these variables in `data()`.